doc/new-private-repository-azure.md

```diff
@@ -0,0 +1,502 @@
+ms.localizationpriority: medium
+---
+
+# Windows Package Mananger private repository
+
+The Windows Package Manager provides a distribution channel for software packages containing their tools and applications. The instructions contained within this document provide guidance on how to setup a private respository that can be connected to using the Windows Package Manager. Providing a comprehensive package manager solution that consists of a command line tool and a set of services for installing applications on Windows 10.
+
+## Automatically create a private repository
+
+A PowerShell script is provided inside of the *.\src\WinGet.RestSource.Infrastructure* folder that will simplify the creation of Azure resources to host your own Windows Package Manager private repository. The Automation script makes use of Azure cmdlets that are only available if the **Az** PowerShell module has been installed. The Automation script will check for the existance of required modules first, and if they are not present will fail and provide guidance on how to import using `Import-Module Az`.
+
+The `automation.ps1` script has the following parameter inputs:
+| Required | Parameter          | Description                                                                                                                |
+|----------|--------------------|----------------------------------------------------------------------------------------------------------------------------|
+| Yes      | ResourcePrefix     | A string of letters which will be prefixed to your newly created Azure resources.                                          |
+| Yes      | Index              | A string of letters or numbers which will be sufixed to your newly created Azure resources.                                |
+| Yes      | AzResourceGroup    | The Resource Group that will be used to contain the Azure resources.                                                       |
+| No       | AzSubscriptionName | The name of the Azure Subscription that will be used to pay for the Azure resources.                                       |
+| No       | AzLocation         | The Azure location where the Azure resources will be created. (Default: westus)                                            |
+| No       | WorkingDirectory   | The folder location that contains this the ARM template files, as well as where the Azure Parameter files will be created. |
+
+The PowerShell script has been configured to work with `Get-Help` providing further details about the script, as well as examples of how to use it.
+
+**How to:**
+
+1. Download a local copy of the Windows Package Manager Rest Source from GitHub (github: [winget-cli-restsource](https://github.com/microsoft/winget-cli-restsource))
+    1. Select *Code* from within GitHub.
+    1. Select *Download ZIP* from the drop-down menu.
+    1. Extract the newly downloaded ZIP file to *C:\Projects\\*
+
+1. Open PowerShell, and run the following command:
```

Comment:
numbering appears broken?

Reply:
Same throughout doc

Reply:
Applying the prefix "1. " to a line will generate a numbered list, auto-incrementing each step.